app/forms/bulk_action_form.rb

```diff
@@ -39,6 +41,11 @@ def csv_as_string
     File.read(csv_file.path)
   end
 
+  def license_options
+    [['-- none --', Constants::NONE_LICENSE_URI]] +
```

Comment:
I don't think we should not set a license in this case.  The consul documentation specifically says not to.  The existing ETDs that do this are an anomaly .

Reply:
changed